### PR TITLE
update slack invite link

### PR DESCRIPTION
**Pull request by ranakan19**
since the current invite link was expired

### CODE REVIEW

File: deploy/templates/notificationtemplates/sandbox/userdeactivated/notification.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
     <p>
         Join the Dev Sandbox community to share your feedback, request extension for your Sandbox environment from the #dev-sandbox channel on DevNation slack workspace.
-        You can join using the following invite - https://dn.dev/DevNationSlack. You can also reach us via email at {{.ReplyTo}} with any questions.
+        You can join using this <a href="https://join.slack.com/t/rhdevnation/shared_invite/zt-27vfuywt3-TC7QuckHfxHatLj6Abk2_A"> invite</a>. You can also reach us via email at {{.ReplyTo}} with any questions.
```
Comment (xcoulon) [inline deploy/templates/notificationtemplates/sandbox/userdeactivated/notification.html:55]:
```suggestion
        You can join using this <a href="https://join.slack.com/t/rhdevnation/shared_invite/zt-27vfuywt3-TC7QuckHfxHatLj6Abk2_A">invite</a>. You can also reach us via email at {{.ReplyTo}} with any questions.
```

File: deploy/templates/notificationtemplates/sandbox/userdeactivating/notification.html [hunk 2]
```diff
@@ -52,7 +52,7 @@
 
     <p>
         Join the Dev Sandbox community to share your feedback, request extension for your Sandbox environment from the #dev-sandbox channel on DevNation slack workspace.
-        You can join using the following invite - https://dn.dev/DevNationSlack. You can also reach us via email at {{.ReplyTo}} with any questions.
+        You can join using this <a href="https://join.slack.com/t/rhdevnation/shared_invite/zt-27vfuywt3-TC7QuckHfxHatLj6Abk2_A"> invite</a>. You can also reach us via email at {{.ReplyTo}} with any questions.
```
Comment (xcoulon) [inline deploy/templates/notificationtemplates/sandbox/userdeactivating/notification.html:55]:
```suggestion
        You can join using this <a href="https://join.slack.com/t/rhdevnation/shared_invite/zt-27vfuywt3-TC7QuckHfxHatLj6Abk2_A">invite</a>. You can also reach us via email at {{.ReplyTo}} with any questions.
```

File: deploy/templates/notificationtemplates/sandbox/userprovisioned/notification.html [hunk 3]
```diff
@@ -56,7 +56,7 @@
 
     <p>
         Join the Dev Sandbox community and engage with the Red Hat team from the #dev-sandbox channel on DevNation slack workspace.
-        You can join using the following invite - https://dn.dev/DevNationSlack. You can also reach us via email at {{.ReplyTo}} with any questions.
+        You can join using this <a href="https://join.slack.com/t/rhdevnation/shared_invite/zt-27vfuywt3-TC7QuckHfxHatLj6Abk2_A"> invite</a>. You can also reach us via email at {{.ReplyTo}} with any questions.
```
Comment (xcoulon) [inline deploy/templates/notificationtemplates/sandbox/userprovisioned/notification.html:59]:
```suggestion
        You can join using this <a href="https://join.slack.com/t/rhdevnation/shared_invite/zt-27vfuywt3-TC7QuckHfxHatLj6Abk2_A">invite</a>. You can also reach us via email at {{.ReplyTo}} with any questions.
```

**Review by xcoulon — APPROVED** [review]
Looks good. I just made some suggestions to remove the extraneous space within the links.

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**